src/main/java/kitchenpos/menus/tobe/domain/menu/service/DefaultProductService.java

```diff
@@ -0,0 +1,38 @@
+package kitchenpos.menus.tobe.domain.menu.service;
+
+import kitchenpos.menus.tobe.domain.menu.vo.MenuProductVO;
+import kitchenpos.menus.tobe.domain.menu.vo.MenuProducts;
+import kitchenpos.products.tobe.infra.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+
+@Service
+public class DefaultProductService implements ProductService{
+
+    private final ProductRepository productRepository;
+
+    public DefaultProductService(ProductRepository productRepository){
+        this.productRepository = productRepository;
+    }
+
+    @Override
+    public MenuProducts findAllPrice (MenuProducts menuProducts){
```

Comment:
```suggestion
    public MenuProducts findAllPrice(MenuProducts menuProducts) {
```
전체적으로 lint가 좀 맞지 않는 것 같은데 IntelliJ의 기능을 활용하셔서 정렬을 자주 해주시거나 Save Action 등 플러그인을 통해 lint를 적용해주시는게 좋을 것 같습니다.